app/helpers/languages_helper.rb

```diff
@@ -189,8 +189,12 @@ module LanguagesHelper
   ISO_639_3 = {
     ast: ['Asturian', 'Asturianu'].freeze,
     ckb: ['Sorani (Kurdish)', 'سۆرانی'].freeze,
+    jbo: ['Lojban', 'Lojban'].freeze,
```

Comment:
The name for Lojban in Lojban is la .lojban.

Reply:
Thank you, I'll fix that.